python/tests/integration/connectors/memory/test_astradb.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import os
+import time
+
+import numpy as np
+import pytest
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.memory.astradb import AstraDBMemoryStore
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+astradb_installed: bool
+try:
+    if os.environ["ASTRADB_INTEGRATION_TEST"]:
```

Comment:
@david-polo how would this env variable get populated? I don't see it present in the `.env.example` that was updated for this PR.

Is there a specific `astradb` package that needs to be added to python's pyproject.toml via a `poetry add <dependency>`? Is the dependency called `astrapy`? 

Couldn't we check if the `ASTRADB_APP_TOKEN` is present?